.github/workflows/release.yml

```diff
@@ -32,9 +32,24 @@ jobs:
     outputs:
       hasWritePermission: ${{ steps.check.outputs.require-result }}
 
+  build-master:
+    name: Build master
+    needs: check-permission
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: Build
+        uses: actions/checkout@master
+        uses: skx/github-action-build@master
+        with:
+          builder: make
```

Comment:
few points here,

- The two steps mentioned above checkouts master branch twice. Instead, we can checkout the branch once and then proceeding to the build step.
 - Build command will run successful even if certain unit tests fail. Should also execute the `validate.sh` script
 - Also should add `if` condition to check if `hasWritePermission` is true

```
 build-master:
    name: Build master
    needs: check-permission
    if: contains(needs.check-permission.outputs.hasWritePermission, 'true')
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@master
        with:
          fetch-depth: 0
      - name: Validate and build
        run: |
          ./validate.sh
          make build

```

Reply:
Thanks for the reivew. Addressed your comments in [9477772](https://github.com/prebid/prebid-server/pull/3032/commits/94777726100c6a4b7b52b7255727957e5011cea4)

> The two steps mentioned above checkouts master branch twice. Instead, we can checkout the branch once and then proceeding to the build step.

Copy/paste error. Corrected

> Build command will run successful even if certain unit tests fail. Should also execute the `validate.sh` script

Good point. Added. I ran the `make` command before `validate` though

> Also should add `if` condition to check if `hasWritePermission` is true

Added